scripts/sample_vcf_to_zarr.py

```diff
@@ -0,0 +1,167 @@
+def main():
+
+    import argparse
+    parser = argparse.ArgumentParser(description="Convert a single sample VCF to Zarr using "
+                                                 "Blosc compression")
+    parser.add_argument("--input",
+                        required=True,
+                        help="path to input VCF file.")
+    parser.add_argument("--output",
+                        required=True,
+                        help="path to output Zarr directory.")
+    parser.add_argument("--sample",
+                        required=True,
+                        help="Sample identifier.")
+    parser.add_argument("--contig",
+                        required=True,
+                        action='append',
+                        dest='contigs',
+                        help="Contig to extract. Multiple values may be provided.")
+    parser.add_argument("--field",
+                        required=True,
+                        action='append',
+                        dest='fields',
+                        help="Field to extract, e.g., 'variants/MQ' or 'calldata/GT'. Multiple "
+                             "values may be provided.")
+    parser.add_argument("--compress-algo",
+                        help="Blosc compression algorithm.  Choose from [zstd, blosclz, lz4, "
+                             "lz4hc, zlib, snappy].",
+                        default="zstd")
+    parser.add_argument("--compress-level",
+                        help="Compression level. Choose integer from [0, 9].",
+                        default=1,
+                        type=int)
+    parser.add_argument("--compress-shuffle",
+                        help="Type of data shuffling used to obtain contiguous runs of same "
+                             "values for improving compression. Choose integer value from "
+                             "NOSHUFFLE (0), SHUFFLE (1), BITSHUFFLE (2) or AUTOSHUFFLE (-1). "
+                             "If -1 (default), bit-shuffle will be used for buffers with "
+                             "itemsize 1, and byte-shuffle will be used otherwise.",
+                        default=-1,
+                        type=int)
+    parser.add_argument("--alt-number",
+                        help="Expected maximum number of alternate alleles.",
+                        default=3,
+                        type=int)
+    parser.add_argument("--tabix",
+                        help="Path to tabix executable v0.2.5+.",
+                        default="tabix")
+    parser.add_argument("--chunk-length",
+                        help="Chunk length in number of variants.",
+                        default=2**18,
+                        type=int)
+    parser.add_argument("--chunk-width",
+                        help="Chunk width in number of samples.",
+                        default=64,
+                        type=int)
+    parser.add_argument("--log",
+                        help="Path to logfile, stdout or stderr. Default: stdout.",
```

Comment:
What do you mean by 'stdout or stderr' - are both streams redirected to an optional provided file here?

Reply:
Yes.  If stdout, then the logs will be written to stdout.  If stderr, then logs will be written to stderr.  If a filepath is given, then the logs will be written to that filepath.